packages/core/src/odata-common/uri-conversion/get-resource-path.ts

```diff
@@ -77,7 +77,7 @@ export function createGetResourcePathForKeys(
 
   function getNullishKeys(keys: Record<string, FieldType>): string[] {
     return Object.entries(keys)
-      .filter(([_, value]) => typeof value === 'undefined' || value === null)
+      .filter(([, value]) => typeof value === 'undefined' || value === null)
```

Comment:
I know this isn't part of this PR, but could you replace this with the `isNullish` function from `@sap-cloud-sdk/util`.
```suggestion
      .filter(([, value]) => isNullish(value))
```

Reply:
@marikaner  has seen the `isNullish` very often in the currency conversion repo ;-)